### PR TITLE
Add GitHub Actions example:Resolves #1749 

### LINKED ISSUE #1749: [Examples] Add GitHub Actions example

### PR DESCRIPTION
**Pull request by shradiphylleia**
Add GitHub Actions example #1749 
created a GitHub actions example to submit tasks that might require GPU.

please let me know if additional changes are needed or additional tasks could be added to make it better. Thanks

### CODE REVIEW

**Comment by shradiphylleia** [comment]
I think the repo has received many spam PR in the last few days and that must have slowed the review process. could a reviewer please let me know if there are any updates or changes I can assist with regarding this PR? 
Thanks

File: examples/GitHub Actions/actions.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+
+
+      # uncomment to install dstack if not available already
```
Comment (SamirPS) [inline examples/GitHub Actions/actions.yml:22]: Why we don't install it by default?

File: examples/GitHub Actions/actions.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  run-dstack-task:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
```
Comment (SamirPS) [inline examples/GitHub Actions/actions.yml:18]: Why 3.10 and not the last version ?

**Review by SamirPS — CHANGES_REQUESTED** [review]

**Comment by peterschmidt85** [comment]
I wish our issues had more context. The thing is, the existing GitHub issues assume that the contributor has at least a basic understanding of how dstack works, its documentation, examples, etc. Issue #1749 is about providing an example on par with other examples – giving users a clear idea of how to use dstack with CI/CD.

The current PR doesn't seem like a useful example. Just compare it to other examples on the website.

**Pull request closed without merging**